crates/xxi-node/src/node/dlc_manager.rs

```diff
@@ -77,7 +77,7 @@ impl<D: BdkStorage, S: TenTenOneStorage + 'static, N: Storage + Sync + Send + 's
             .on_dlc_message(&message.into(), to_secp_pk_29(node_id))?;
 
         let response = match response {
-            Some(response) => Some(TenTenOneMessage::try_from(response)?),
+            Some(response) => Some(TenTenOneMessage::build_from_response(response)?),
```

Comment:
🔧 It could be a `From` since it's infallible now 🙂

Reply:
I don't think its a good idea, because a `From` implementation would suggest it can transform any `Message` into a `TenTenOneMessage` which is not true. This only works because we know that we are dealing with a response which can never be an offer. I feel this context would get lost in a `From` implementation.

Also, if we were to enhance further messages, besides the `Offer` messages, this `From` implementation may need more arguments to transform a `Message` into a `TenTenOneMessage`.

Reply:
You are right; I am wrong.

😛